pkg/sql/catalog/systemschema/system.go

```diff
@@ -669,7 +669,7 @@ CREATE TABLE system.statement_statistics (
 			total_estimated_execution_time,
 			p99_latency
 		)
-)
+) WITH (sql_stats_automatic_collection_fraction_stale_rows = 0.9)
```

Comment:
It's fine for this number to go above 1.0. Based on some [back-of-the-envelope math](https://github.com/cockroachdb/cockroach/issues/147443#issuecomment-2932501220) I was thinking 4.0, but if we want to be more conservative, that's fine. It might be nice to pick an even multiple of 0.2 (the default).

Whatever we pick, we should also set `sql_stats_automatic_partial_collection_fraction_stale_rows` to 1/4 of it.